src/webgpu/web-platform/copyImageBitmapToTexture.spec.ts

```diff
@@ -129,27 +254,42 @@ g.test('from_ImageData')
       .combine(poptions('height', [1, 2, 4, 15, 255, 256]))
       .combine(poptions('alpha', ['none', 'premultiply']))
       .combine(poptions('orientation', ['none', 'flipY']))
+      .combine(poptions('inputColorFormat', ['rgba8unorm'] as const))
```

Comment:
I added this because there should be bgra8unorm and rgba16float input format but after reconsidering this again, I think you're right. For current imageData to imageBitmap, I don't know how to make imageBitmap and store the input format as other formats. 
I'll remove this.

Reply:
Done.